### DISCUSSION TITLE
Welcome to ScandiPWA Discussions!

### DISCUSSION THREAD

**Discussion by lianastaskevica** [scandipwa/scandipwa#1834] [General]
### :pray: Q&A 

Have a question about theming or plugin creation? 

&rarr; leave your question [here](https://github.com/scandipwa/scandipwa/discussions?discussions_q=category%3AQ%26A)

### :bulb: Ideas

Know which feature we should do next?

&rarr; leave your idea [here](https://github.com/scandipwa/scandipwa/discussions?discussions_q=category%3AIdeas)

### :rocket: Install & Upgrade

Have troubles with installation or upgrade? 

&rarr; leave your question [here](https://github.com/scandipwa/scandipwa/discussions?discussions_q=category%3A%22Install+%26+Upgrade%22)

### :raised_hands: Show and tell

Resolved a challenging issue or implemented a cool feature? 

&rarr; share your experience [here](https://github.com/scandipwa/scandipwa/discussions?discussions_q=category%3A%22Show+and+tell%22)

### :hash: General

Want to just chat with ScandiPWA community? 

&rarr; say hello [here](https://github.com/scandipwa/scandipwa/discussions?discussions_q=category%3AGeneral)

**Comment by ukashif2** [comment]
Can anybody answer?

Namespace not defined while running php bin/magento scandipwa:theme:bootstrap can anyone help Magento version is 2.3.6
